docs/resources/resources-glossary.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Office Add-ins glossary of terms
+description: A glossary of terms commonly used throughout the Office Add-ins documentation.
+ms.date: 02/10/2022
+ms.localizationpriority: medium
+---
+
+# Office Add-ins glossary
+
+This is a glossary of terms commonly used throughout the Office Add-ins documentation.
+
+## add-in
+
+Office Add-ins are web applications that extend Office applications. These web applications add new functionality to the Office application, such as bring in external data, automate processes, or embed interactive objects in Office documents.
+
+Office Add-ins differ from VBA, COM, and VSTO add-ins because they offer cross-platform support (web, Windows, Mac, and iPad) and are based on standard web technologies (HTML, CSS, and JavaScript). The primary programming language of an Office Add-in is JavaScript or TypeScript.
+
+## add-in commands
+
+**Add-in commands** are UI elements, such as buttons and menus, that extend the Office UI for your add-in. When users select an add-in command element, they initiate actions such as running JavaScript code or displaying the add-in in a task pane. Add-in commands let your add-in look and feel like a part of Office, which gives users more confidence in your add-in. See [Add-in commands for Excel, PowerPoint, and Word](../design/add-in-commands.md) and [Add-in commands for Outlook](../outlook/add-in-commands-for-outlook.md) to learn more.
+
+See also: [ribbon, ribbon button](#ribbon-ribbon-button).
+
+## application
+
+**Application** refers to an Office application. The Office applications that support Office Add-ins are Excel, OneNote, Outlook, PowerPoint, Project, Visio, and Word.
```

Comment:
I don't think Visio technically supports add-ins. The Visio JS APIs allow for Visio diagrams to be embedded in SharePoint.

Reply:
My inclination is to simply remove Visio from this list. Do you think that's reasonable?